doc/src/manual/noteworthy-differences.md

```diff
@@ -70,7 +70,7 @@ may trip up Julia users accustomed to MATLAB:
     in an interactive session. In Julia, unlike MATLAB, `ans` is not set when Julia code is run in
     non-interactive mode.
   * Julia's `struct`s do not support dynamically adding fields at runtime, unlike MATLAB's `class`es.
-    Instead, use a [`Dict`](@ref).
+    Instead, use a [`Dict`](@ref). Dict in Julia isn't ordered.
```

Comment:
Seem unrelated to the PR?

Reply:
Dict unordered surprised a colleague of mine, as similar in MATLAB is ordered. Github wants small titles.